_config.yml

```diff
@@ -50,5 +50,10 @@ compress_html:
   comments: all
   endings: all
   startings: []
-  blank_lines: false
+  blanklines: true # required for Mathjax
```

Comment:
Since Mathjax is a custom script, I'd like to separate out this fix (changing `blank_lines` -> `blanklines`) as its own PR.

Mathjax support should added by users as custom behavior to this theme, not included as a dependency of this theme.

Reply:
OK

Reply:
Should I close this PR and open a new one for the fix, or will you simply make the fix yourself?

Reply:
You can modify this PR if you like